src/main/java/uk/gov/ons/ctp/response/action/scheduled/distribution/ActionDistributor.java

```diff
@@ -86,29 +81,47 @@ public DistributionInfo distribute() {
             .build();
 
     try {
-      final List<Action> actions = retrieveActions(actionType);
+      List<Action> actions = retrieveActions(actionType);
       processActions(actions, requestCount, cancelCount);
-    } catch (final Exception e) {
-      log.with("action_type", actionType.getName()).error("Failed to process action type", e);
+    } catch (Exception e) {
+      log.with("action_type", actionType).error("Failed to process", e);
```

Comment:
I'm worried about this... catching all exceptions like this means we never rollback, which means we might not retry properly in the event of a temporary glitch - we're not letting things unwind themselves elegantly